Formula/s/smake.rb

```diff
@@ -1,8 +1,8 @@
 class Smake < Formula
   desc "Portable make program with automake features"
-  homepage "https://s-make.sourceforge.net/"
+  homepage "https://codeberg.org/schilytools/schilytools"
   url "https://codeberg.org/schilytools/schilytools/archive/2023-09-28.tar.gz"
-  version "1.7-2023-09-28"
```

Comment:
This is less about the `version`/`livecheck`, but more about the project situation (per previous discussions, the project is more in the maintenance mode and 1.7 is more for the historical recognition)

Reply:
Not sure I understand, do you have a concern about removing the `1.7-` from the version?

Many of these utilities used to be distributed separately but now development/maintenance is moving into one repo, which is following a date-based version scheme.

Reply:
I understand that [1.7 is the `smake` tool version](https://codeberg.org/schilytools/schilytools/src/commit/20e082e7ef865caed098aeef43405c1700635dff/smake/version.h) and historically these tools used a release version scheme like 1.2.3 (i.e., in the prior individual SourceForge projects).

Under the new maintenance [on Codeberg], upstream is simply using the schilytools bundle version (e.g., 2023-09-28) for versioning. Upstream has [suggested using the tool version or the date of the schilytools bundle for versioning](https://codeberg.org/schilytools/schilytools/issues/61) but also said that using a compound version (like 1.7-2023-09-28) is fine, too.

As discussed in https://github.com/Homebrew/homebrew-core/pull/149240, upstream sometimes makes changes to a tool (e.g., `star`) without updating the tool version or date (e.g., in `version.h`/`version.c`). With that in mind, the schilytools bundle/release date is what's meaningful for versioning and the tool version doesn't do much for us in terms of maintaining the formula.

Rather, including the tool version in the formula `version` makes these formulae harder to maintain. The tool version isn't apparent from the release information or Git tags, so it's necessary to check the related file for a given release tag to find the tool version. This leads to a lengthy and involved `livecheck` block, as seen here: https://github.com/Homebrew/homebrew-core/pull/149240#discussion_r1344179971

Besides that, as mentioned in the `star` PR, the shilytools formulae will all be using a release bundle but if we include the tool version in the formula `version`, we have to individually bump each of these formulae. If these formulae simply using the upstream schilytools bundle/release date as their version, then we can add them to `synced_versions_formulae.json` and that will help to keep all of these up to date.

Besides that, the upstream maintainer isn't using the tool version in the [version of the FreeBSD port (`2023.09.28`)](https://www.freshports.org/devel/smake) they help to maintain, so the tool version doesn't seem to be very important from their (and our) standpoint.

Does that help to explain things?

Reply:
@samford thanks for the write up, i am fine with this update.